### PR TITLE
Fixes #37154 - run componentRegistry.registerMultiple once

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/componentRegistry.js [hunk 1]
```diff
@@ -182,6 +182,8 @@ const coreComponets = [
   { name: 'WelcomeArchitecture', type: WelcomeArchitecture },
 ];
 
-componentRegistry.registerMultiple(coreComponets);
+if (!componentRegistry.registry[coreComponets[0].name]) {
+  componentRegistry.registerMultiple(coreComponets);
+}
```
Comment (jeremylenz) [inline webpack/assets/javascripts/react_app/components/componentRegistry.js]: Can we use this opportunity to finally add "n" to `coreComponents`? :innocent:

**Review by jeremylenz — COMMENTED** [review]
Removes the "component name already taken" errors for me :+1:

File: webpack/assets/javascripts/react_app/components/componentRegistry.js (hunk 1 above)
Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/componentRegistry.js]: lol sure :)

**Review by jeremylenz — APPROVED** [review]
Katello React failures unrelated. thanks @MariaAga !

**Pull request merged**